catalog/registry/catalog-registry-common/src/main/java/ddf/catalog/registry/common/metacard/RegistryObjectMetacardType.java

```diff
@@ -27,8 +27,6 @@
  */
 public class RegistryObjectMetacardType extends MetacardTypeImpl {
 
-    public static final String REGISTRY_TAG = "registry";
-
     public static final String REGISTRY_METACARD_TYPE_NAME = "registry";
```

Comment:
Remove REGISTRY_METACARD_TYPE_NAME and use the REGISTRY_TAG from RegistryConstants